release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -39,23 +39,23 @@ For each release, the schedule with deliverables is added to the release directo
     - [Inform localization teams](#inform-localization-teams)
     - [Request for temporary write access to website repo](#request-for-temporary-write-access-to-website-repo)
     - [Freeze Kubernetes website](#freeze-kubernetes-website)
+    - [Merge `master`](#merge-master)
     - [Review milestones](#review-milestones)
 - [Release Day](#release-day)
-    - [Merge `master`](#merge-master)
+    - [Merge the integration branch](#merge-the-integration-branch)
+    - [Publish the release blog post](#publish-the-release-blog-post)
     - [Create release with tag](#create-release-with-tag)
     - [Unfreeze](#unfreeze)
     - [Close the [future release] milestone](#close-the-future-release-milestone)
 - [Post Release Verification, Cleanup, and Handoff](#post-release-verification-cleanup-and-handoff)
-    - [Update Release Notes Changelog](#update-release-notes-changelog)
```

Comment:
This step is no longer required?

Reply:
It is required but is covered by L:709.
This was a duplicate that I believe I can remove.

Reply:
ah, I see. Thanks for clarification.